main.go

```diff
@@ -137,7 +138,12 @@ func main() {
 	}
 
 	okClientProvider := okteto.NewOktetoClientProvider()
+	k8sClientProvider := okteto.NewK8sClientProvider()
 	at := analytics.NewAnalyticsTracker()
+	projectRoot, err := os.Getwd()
+	if err != nil {
```

Comment:
should we fail? logic around get repo url using this variable is used and Im not sure if it will works